modules/tinymce/src/core/main/ts/Rtc.ts

```diff
@@ -68,6 +73,11 @@ interface RtcRuntimeApi {
   };
 }
 
+interface RTCExtra {
```

Comment:
Wouldn't this share the same type as the UndoManagerTypes Extra you would want all 4 variants here as well?

Reply:
Probably. I think I didn't here due to the linting protesting and I never went back to try and change it. I can do that.